snapcraft/sources.py

```diff
@@ -0,0 +1,26 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Legacy support for local plugins."""
+
+import sys as _sys
+
+if _sys.platform == "linux":
+    from snapcraft_legacy.sources import get
```

Comment:
this feels dangerous, is this how we do the other redirects for v1/v2 plugins?

Reply:
It is, see https://github.com/snapcore/snapcraft/blob/hotfix/7.5/snapcraft/plugins/v1/__init__.py

Reply:
this was mostly a backwards compatibility no change _hack_